### ISSUE TITLE
text on top of my editor

### ISSUE THREAD

**Issue by Omzig** [ironmansoftware/powershell-universal#3883]
### Version

5.0.10

### Severity

Low

### Environment

Nested IIS

### Steps to Reproduce

add an api
edit it

### Expected behavior

```console
nothing in the way of the editor
```

### Actual behavior

```console
copyright on top of my code
```

### Screenshots/Animations

![image](https://github.com/user-attachments/assets/119c8a2c-71e0-42da-b39d-b457f35df4e8)

**Comment by adamdriscoll** [comment]
Closing as you should just get a bigger monitor.

**Comment by adamdriscoll** [comment]
Just kidding. It should be fixed in 5.0.11. 😎
